Server/EncryptionService/main.py

```
@@ -0,0 +1,358 @@
+
+
+# Stats endpoint for Admin service
+@app.get("/stats")
+async def get_service_stats():
+    """
+    Get current statistics for the encryption service (RPM, RPD, errors, latency).
+    Used by Admin service for monitoring.
+    """
+    try:
+        stats = get_stats("encryption")
+        return stats
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)